### ISSUE TITLE
Ability to expose postgres service

### ISSUE THREAD

**Issue by iloveitaly** [aaronstillwell/terraform-provider-dokku#18]
**Describe the solution you'd like**
Would love the ability to `expose = true` or `expose = 1024` on a postgres + redis service

**Comment by aaronstillwell** [comment]
@iloveitaly #20 adds an `exposed` attribute that takes a host:port string e.g

```hcl
resource "dokku_postgres_service" "my-pg" {
  name = "my-pg"
  exposed = "0.0.0.0:8525"
}
```

Does that suffice for your use case?

**Comment by iloveitaly** [comment]
Looks amazing!!

**Comment by aaronstillwell** [comment]
This is included in v0.5.0 under `expose_on`

**Comment by iloveitaly** [comment]
Amazing! Thank you!
